### DISCUSSION TITLE
Math simbols and equalition writing in outline

### DISCUSSION THREAD

**Discussion by victordariovera** [outline/outline#2073] [Ideas]
Hi!

I'm trying Outline to document all my University stuff (Computer Cs.) and I don't get a way for writing math equaltions. I dont know if its a limitation of markdown, that I know its the undeliying format for Outline documents. There is a way to implement this?

What kind of library uses Outline for manage markdown? Maybe I can colaborate with code and implement math writing as a native feature.

**Comment by ghost** [comment]
I'd like this feature too, it is quite unpleasant to write documents with math equations inside.

**Reply by zhixingfeng** [reply]
I definitely agree with you that Outline should include math equations in their editor. I'm trying to build a science lab knowledge base, which really need math equations. 

BTW, I don't think math is a limitation of Markdown. Tried math like $\frac{X}{Y}$ in Markdown using VS code, worked perfectly. When I copied it into outline, the equation was not rendered.
